org.sugarlabs.Pippy.json

```diff
@@ -242,6 +63,21 @@
                 }
             ]
         },
+        {
+            "name" : "vte",
+            "buildsystem" : "meson",
+            "build-options" : {
+                "env" : {
+                    "CXXFLAGS" : "-fno-exceptions"
+                }
+            },
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/vte.git"
```

Comment:
I don't think it's a good idea to build vte from master

Reply:
Ouch~ I just merged~ it... if there's a way you can cancel it, I can fix this :)

Reply:
You can stop the build from here https://flathub.org/builds/#/apps/org.sugarlabs.Pippy

Reply:
Done, thanks! anything else you can spot?